doc: update some dynamic parameters from acrn-dm usage (#7041)
projectacrn/acrn-hypervisor: doc/reference/config-options-launch.rst

  ``[@]stdio|tty|pty|sock:portname[=portpath][,[@]stdio|tty|pty:portname[=portpath]]``.

``cpu_affinity``:

>   List of Service VM vCPUs assigned to this VM, the Service VM vCPUs are identified

Reviewer: ```suggestion
  A comma-separated list of Service VM vCPUs assigned to this VM. A Service VM vCPU is identified
```

Author: Updated.

>   by their lapic IDs.

Reviewer: ```suggestion
  by its lapic ID.
```

Author: Updated.
